Corona-Warn-App/src/deviceForTesters/res/layout/fragment_test_datadonation.xml

```diff
@@ -115,6 +115,98 @@
                     app:layout_constraintTop_toBottomOf="@id/safetynet_requirements_body" />
             </androidx.constraintlayout.widget.ConstraintLayout>
 
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/last_analytics_container"
+                style="@style/Card"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/spacing_tiny">
+
+                <TextView
+                    android:id="@+id/analytics_last_submit_title"
+                    style="@style/headline6"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginEnd="8dp"
+                    android:text="Last Analytics Submission"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <TextView
+                    android:id="@+id/analytics_last_submit_body"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_tiny"
+                    android:textIsSelectable="true"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/analytics_last_submit_title"
+                    tools:text="Body" />
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
please check for layout Stacking (constraint->constraint->...)
I'm not sure here

Reply:
i'm too stupid to do it without as the stacked layout also contains a card border